### ISSUE TITLE
The first tab element is displaced vertically

### ISSUE THREAD

**Issue by micobarac** [jgthms/bulma#2172]
This is about **Bulma**.

### Overview of the problem

This is about the Bulma **CSS framework**.

I am using the latest Bulma version.

### Description

The first tab element is displaced vertically a few pixels, forming a small gap from the bottom border. This problem is almost 2 years old, but wasn't fixed along the way.

https://github.com/jgthms/bulma/issues/474

### Steps to Reproduce

I am facing the this issue on Windows 10 x64 with the latest Chrome version.

![bug](https://user-images.githubusercontent.com/16001837/47269287-fa20e480-d55b-11e8-95b4-816937a5784b.png)

```html
<section class="section content">
  <div class="container is-fluid">
    <h1 class="title">Dashboard</h1>
    <div class="tabs">
      <ul>
        <li class="is-active"><a>Pictures</a></li>
        <li><a>Music</a></li>
        <li><a>Videos</a></li>
        <li><a>Documents</a></li>
      </ul>
    </div>
  </div>
</section>
```

 Aligning items to the baseline did the trick.

```sass
.tabs ul { align-items: baseline; ... }
```

![ok](https://user-images.githubusercontent.com/16001837/47269324-600d6c00-d55c-11e8-91cd-88dde55cc4a7.png)

### Expected behavior

All tabs should be aligned vetically.

### Actual behavior

The first tab is displaced.

**Comment by mobayen** [comment]
same here
this style causes 
```css
.content li + li {
    margin-top: 0.25em;
}
```
either removeing it or lose the "+ li" part will solve the issue

**Comment by MikelisB** [comment]
If it helps resolving it - same to me and the fix by OP did the trick. Thanks! :)

**Comment by logological** [comment]
The issue appears to have been fixed in Bulma 0.7.5.  (I can reproduce it with Bulma 0.7.4, but not Bulma 0.7.5.)  I think this issue can be closed now.
